libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt

```diff
@@ -321,12 +231,18 @@ class MapboxRouteLineView(var options: MapboxRouteLineOptions) {
      * @param style an instance of the Style
      */
     fun hideOriginAndDestinationPoints(style: Style) {
-        updateLayerVisibility(style, RouteLayerConstants.WAYPOINT_LAYER_ID, Visibility.NONE)
+        hideLayer(style, RouteLayerConstants.WAYPOINT_LAYER_ID)
+    }
+
+    private fun showLayer(style: Style, layerId: String) {
```

Comment:
The show and hide methods you added are almost entirely duplicated code. That's the reason there was a single method with a visibility parameter so that there wouldn't be duplicated code.

Reply:
what about single responsibility?
it's easier to read code with `show/hide` methods than with `updateVisibility`.
and no need to pass Visible param

Reply:
The update visibility method had the single responsibility of changing the visibility.  That's all it did. The only difference between the two methods you created is the visibility parameter. DRY.